server/PositivePixelCount/PositivePixelCount.xml

```diff
@@ -67,8 +74,9 @@
       <description>left,top,width,height of the region of interest.  All -1 means the whole image is used.</description>
       <default>-1,-1,-1,-1</default>
     </region>
-    <image>
-      <longflag>outputLabelImage</longflag>
+    <image fileExtensions='.png'>
+      <name>outputLabelImage</name>
+      <index>1</index>
```

Comment:
This will make it a mandatory output parameter. When it is run on the entire slide we cant produce this output image as there is no support for WSI writing in large image.